pandas/core/series.py

```diff
@@ -1429,8 +1429,28 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algorithms.mode(self)
 
-    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
     def unique(self):
+        """
+        Return unique values in the object. Uniques are returned in order
```

Comment:
This should be a single line. Can you simplify? You can use an extended summary for the bit about uniques.

Reply:
broken into two lines. and minor rewording.